source/mesh_deformation/interface.cc

```diff
@@ -282,6 +310,20 @@ namespace aspect
       }
       prm.leave_subsection ();
 
+      // Remove all prescribed mesh deformation boundary indicators from the
+      // set of tangential mesh deformation boundary indicators
+      for (const auto &boundary_id : mesh_deformation_boundary_indicators_set)
+        tangential_mesh_boundary_indicators.erase(boundary_id);
+
+      // Add all periodic boundaries to the set of tangential mesh deformation
+      // boundary indicators
```

Comment:
The comment says what you are doing, but not why. Would you be willing to add a sentence (here and above)?